app/controllers/cases/case_transitions_controller.rb

```diff
@@ -0,0 +1,61 @@
+module Cases
+  class CaseTransitionsController < CasesController
```

Comment:
Good point - it made sense when there was a 'create' action instead of these very specific offender-sar actions, but not now! I've moved these into `offender_sar_controller.rb`